vscode-dotnet-runtime-library/distro-data/distro-support.json

```diff
@@ -0,0 +1,90 @@
+{
+	"Ubuntu": {
```

Comment:
We should consider it (and that was actually the initial proposal.) I prefer this method because it simplifies the number of files we need to webpack, make sure permissions are correct for, etc. Review should be siloed to certain sections of the file in general, I would hope diff tools are smart enough to work ok in this context. I think you could go either way though!

As for 2) I would be glad to make that. @baronfel may you share where that would go, or what it would look like? Is there a documentation page we'd need to link to it, and is it just a doc itself?

Reply:
JSON Schema files are themselves JSON (there's a really good tutorial [here](https://json-schema.org/learn/getting-started-step-by-step.html) on writing them), and it can just live in this repo somewhere. In the repo settings (or in this json file itself), you can associate a file to a schema. The most IDE-agnostic way to do this association is to add a `"$schema"` key to the `distro-support.json` file pointing to the JSON Schema itself, local in this repo. Happy to take a stab at an initial one as an example, if that would help!

Reply:
Thank you Chet :) I added a json schema file like you said with the schema key.

Reply:
That schema file is a thing of beauty.